src/features/landing/components/ForWhoSection/MainTitle/MainTitle.stories.tsx

```
@@ -0,0 +1,18 @@
+import { Meta, StoryObj } from '@storybook/react';
+
+import MainTitle from './MainTitle';
+
+const meta: Meta<typeof MainTitle> = {
+  title: 'Components/ForWhoSection/MainTitle',  
+  component: MainTitle,
+  tags: ['autodocs'],
+  parameters: {
+    layout: "",  
```

ESLint

[error] Strings must use singlequote.

(quotes)